types/ckeditor__ckeditor5-utils/src/emittermixin.d.ts

```diff
@@ -0,0 +1,110 @@
+import EventInfo from "@ckeditor/ckeditor5-utils/src/eventinfo";
+import { PriorityString } from "@ckeditor/ckeditor5-utils/src/priorities";
```

Comment:
```suggestion
import EventInfo from "./eventinfo";
import { PriorityString } from "./priorities";
```
should those be relative?